packages/block-editor/src/components/inserter/media-tab/api.js

```diff
@@ -0,0 +1,183 @@
+/* eslint no-console: [ 'error', { allow: [ 'error', 'warn' ] } ] */
+/**
+ * WordPress dependencies
+ */
+import { dispatch, select } from '@wordpress/data';
```

Comment:
I know there are precedents but I don't like APIs that rely on global dispatch/select because you don't really know which "block-editor" you're actually updating. I think this API should just be a regular action.